components/org.wso2.micro.integrator.initializer/src/main/java/org/wso2/micro/integrator/initializer/dashboard/HeartBeatComponent.java

```diff
@@ -205,7 +206,13 @@ public static boolean isDashboardConfigured() {
 
     public static JsonObject getJsonResponse(CloseableHttpResponse response) {
         String stringResponse = getStringResponse(response);
-        return new JsonParser().parse(stringResponse).getAsJsonObject();
+        JsonObject responseObject = null;
+        try {
+            responseObject = new JsonParser().parse(stringResponse).getAsJsonObject();
+        } catch (JsonParseException e) {
+            log.error("Error occurred while parsing the heartbeat response.", e);
```

Comment:
any possibility of changing this log level to debug as well?

Reply:
It will be changed by https://github.com/wso2/micro-integrator/pull/2827